src/_includes/cwl/custom-functions.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Capitalize each word passed. Will split the text by spaces.
+ * @param {String} message - The input message.
+ * @return {String} the message with each word with its initial letter capitalized.
+ */
+function capitalizeWords (message) {
+  if (message === undefined || message === null || typeof message !== 'string' || message.trim().length === 0) {
+    return '';
+  }
+  return message
+    .split(' ')
+    .map(function (token) {
+      return token.charAt(0).toUpperCase() + token.slice(1);
+    })
```

Comment:
This was the simplest JS code I could think of, that's actually common to see in code bases (different than fibonacci sequence or factorial) as it's not provided by JavaScript out of the box as in Python with `'string'.capitalize()`. Also avoided anything random, so that the output of the examples is more deterministic (which I intend to make even more deterministic by removing user name, directories, etc. with a container + Sphinx directive… for another issue).